src/hooks/usePoolData.ts

```diff
@@ -312,32 +310,19 @@ export default function usePoolData(
       }
 
       const metaSwapAddress = POOL.metaSwapAddresses?.[chainId]
+      const poolAddressToCheckMigrationStatus = metaSwapAddress || poolAddress
+      const migrationAddress = metaSwapAddress ? metaSwapAddress : poolAddress
```

Comment:
is this condition safe to assume?

Reply:
This is identical to `metaSwapAddress || poolAddress` above as `metaSwapAddress` will be falsey if undefined

Reply:
til